src/control/cmd/daos_server/README.md

```diff
@@ -1,67 +1,60 @@
 # DAOS Server
 
-The DAOS Server (control-plane) can be run either as a foreground
-process to issue hardware provisioning commands or as a background
-process (binary name followed by the "start" subcommand) when it will
-listen for instructions from the
-[management tool](/src/control/cmd/dmg/README.md) and manage bring-up
-of the DAOS system.  A DAOS Server instance will typically be run on
-every storage node to orchestrate the operation of the managed DAOS
-Engine processes.
+The DAOS Server (control-plane) can be run either as a foreground process to
```

Comment:
(minor nit) The hyphenation of control/data plane in this file is inconsistent with the wording in src/control/README.md